sgx-revm.manifest.template

```diff
@@ -15,7 +14,7 @@ loader.env.RUST_BACKTRACE = "full"
 fs.mounts = [
   { path = "/lib", uri = "file:{{ gramine.runtimedir() }}" },
   { path = "{{ arch_libdir }}", uri = "file:{{ arch_libdir }}" },
-
+  { path = "/etc", uri = "file:/etc" },
```

Comment:
Should we change this to the exact file that you need instead of mounting the whole etc inside?

Reply:
That's a good question.
We have the certificates file in trusted files, so I think we could actually remove this mount (but I didn't test that)
What would be even better would be to remove the certificates file from trusted files and mount it here

I'm not merging this PR until this is thought through, but I'm not versed enough with gramine to do this quickly - I'd appreciate help

Reply:
Sure, I can try and help with testing that.
Why do you think it is better to mounting it here instead of having it in trusted files? wouldn't it be the same result at the end? i.e. mounting would simply mount the file in and grant trusted access to it and in both cases it would be part of the measurement or am I wrong?